### PR TITLE
cluster-mgmt: update discovery doc for etcd2

### PR DESCRIPTION
**Pull request by robszumski**
Add info for etcd2

cc @xiang90

### CODE REVIEW

File: cluster-management/setup/cluster-discovery/index.md [hunk 1]
```diff
@@ -49,19 +49,21 @@ Specific documentation are provided for each platform's guide. Not all providers
...
-4. Machine 2 boots and submits its `peer-addr` address `10.10.10.2`. It also reads back the list of existing peers (only `10.10.10.1`) and attempts to connect to the address listed.
+4. Machine 2 boots and submits its `-initial-advertise-peer-urls` address `10.10.10.2`. It also reads back the list of existing peers (only `10.10.10.1`) and attempts to connect to the address listed.
 5. Machine 2 connects to Machine 1 and is now part of the cluster as a follower.
-6. Machine 3 boots and submits its `peer-addr` address `10.10.10.3`. It reads back the list of peers ( `10.10.10.1` and `10.10.10.2`) and selects one of the addresses to try first. When it connects to a machine in the cluster, the machine is given a full list of the existing other members of the cluster.
+6. Machine 3 boots and submits its `-initial-advertise-peer-urls` address `10.10.10.3`. It reads back the list of peers ( `10.10.10.1` and `10.10.10.2`) and selects one of the addresses to try first. When it connects to a machine in the cluster, the machine is given a full list of the existing other members of the cluster.
```
Comment (crawford) [inline cluster-management/setup/cluster-discovery/index.md]: "( `10.10.10.1`" -> "(`10.10.10.1`"

**Comment by crawford** [comment]
Otherwise, LGTM.

File: cluster-management/setup/cluster-discovery/index.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Boot each one of the machines with identical cloud-config and they should be aut
 #cloud-config
 
 coreos:
-  etcd:
+  etcd2:
```
Comment (xiang90) [inline cluster-management/setup/cluster-discovery/index.md]: sorry for the delay... (I was debugging etcd)

We still need to change line 39 to etcd2.service and line 36-37 to use new flags.

Comment (xiang90) [inline cluster-management/setup/cluster-discovery/index.md]: https://coreos.com/docs/running-coreos/cloud-providers/digitalocean/ this page is correct though.

Comment (robszumski) [inline cluster-management/setup/cluster-discovery/index.md]: Whoops, don't know how I missed this

**Pull request merged**